nep-7.mediawiki

```diff
@@ -0,0 +1,117 @@
+<pre>
+  NEP: 7.0
```

Comment:
The number `7` has been alloced for [NEP: Triggers for NeoContract](https://github.com/neo-project/proposals/pull/15).
After a proposal being accepted, I will alloc a number for it.